### ISSUE TITLE
NIST Pages Project Logo

### ISSUE THREAD

**Issue by bbattaglini** [usnistgov/OSCAL#296]
# User Story: 
  
As an OSCAL website content owner, I want the website to utilize a project logo so the target audience can easily identify the project and site assets. 
  
## Goals: 
Reach core team consensus on the OSCAL project logo. 
Establish the project logo for the OSCAL project that can be applied to the website. 
  
## Dependencies: 
Issue #279 is set as an Epic for this user story in ZenHub. To view the hierarchy, see: https://app.zenhub.com/workspaces/oscal-5b4e5f99c417015bdfbefa03/boards?repos=68406934 

Issue #294  is set as an Epic for this user story in ZenHub. To view the hierarchy, see: https://app.zenhub.com/workspaces/oscal-5b4e5f99c417015bdfbefa03/boards?repos=68406934 

## Acceptance Criteria 
Using the site map, review and confirm that the OSCAL project logo is incorporated into the Landing Page. 
Using the site map, review and confirm that the OSCAL project logo is incorporated into the Get Involved Page. 
Using the site map, review and confirm that the OSCAL project logo is incorporated the Content Pages.

**Comment by iMichaela** [comment]
COntacted NISTDo (Natasha Hanacek) for help with the logo

**Comment by howieavp76** [comment]
@iMichaela - started a campaign on 99Designs to crowdsource ideas from designers.  You should get many options to choose from in the next week which I will share here.  Once NIST selects their favorite, there is the opportunity to iterate on that design and bring in the appropriate people to finalize it.

I have always found this to be a fun and collaborative process.  Look forward to sharing the designs with the team as they come in.

**Comment by howieavp76** [comment]
@iMichaela  - 19 design options submitted so far, I can demo this during the sprint review tomorrow
<img width="880" alt="screen shot 2019-02-27 at 3 09 44 pm" src="https://user-images.githubusercontent.com/33614714/53520679-303f1980-3aa4-11e9-9623-46f7f9fc3b4f.png">

**Comment by iMichaela** [comment]
@howieavp76 - thank you. The problem we have is coming from DoC's policy that requires special approval for logos. I already escalated this problem.

**Comment by howieavp76** [comment]
@iMichaela - got it.  If you are able to get approval, we will have a large number of designs for the OSCAL team to review.  Some of them already look pretty good :)

**Comment by howieavp76** [comment]
We are up to 88 designs now.  https://99designs.com/logo-design/contests/oscal-logo-design-896261/entries

**Comment by howieavp76** [comment]
@iMichaela - we ended up with 152 designs.  We need to select the finalists and provide feedback either today or tomorrow.  From there, they will give us more options based on our feedback and we can select the one we want.

**Comment by howieavp76** [comment]
Reviewed 152 logos with @iMichaela on Wednesday afternoon.  Down-selected to 6 finalists and provided guidance for the next round.  Final designs will come back in the next 72 hours.

**Comment by howieavp76** [comment]
https://99designs.com/logo-design/contests/oscal-logo-design-896261/entries

**Comment by brianrufgsa** [comment]
3/7/2019: Travis and Michaela have made great progress with this. They are down-selecting six from the latest list, and will put them up to poll the team for preferences. 
Once the team has selected something, it must be approved by DoC via NIST Public Affairs.

**Comment by howieavp76** [comment]
@iMichaela - final designs are in, we can review at your convenience to select the winner for the logo and close this issue.

**Comment by howieavp76** [comment]
The two favorites for the logo are below.  Email will be sent asking for the team's vote to select the final one.
![Screen Shot 2019-03-18 at 10 44 13 AM](https://user-images.githubusercontent.com/33614714/54538615-f0fd3d80-496a-11e9-9306-79509722c4fb.png)

**Comment by howieavp76** [comment]
Link to all of the finalists are below:

https://99designs.com/logo-design/contests/oscal-logo-design-896261/entries?filter=three_to_five_stars

**Comment by howieavp76** [comment]
Waiting on input until COB today (March 18 @ 6 PM) then we will notify the winner.

**Comment by iMichaela** [comment]
# 3/21/2019 # 
We have a winner - see above. The approval process for the logo will be captured in a separate issue
